vowpalwabbit/interactions_predict.h

```diff
@@ -71,9 +78,42 @@ inline float INTERACTION_VALUE(float value1, float value2) { return value1 * val
 
 // #define GEN_INTER_LOOP
 
+std::tuple<std::pair<features::const_audit_iterator, features::const_audit_iterator>,
```

Comment:
I know its kind of bad practice, but should we typedef this nasty return type for readability's sake?